code/modules/research/techweb/all_nodes.dm

```diff
@@ -230,6 +230,15 @@
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
 	export_price = 5000
 
+/datum/techweb_node/regulated_bluespace
+	id = "regulated_bluespace"
+	display_name = "Regulated Bluespace Research"
+	description = "Bluespace technology using stable and balanced procedures, unwilling to damaging the fabric of bluespace. Lawed by galactic conventions."
```

Comment:
This reads quite poorly, not your fault though since it seems english isn't your primary language. You cant just replace a bunch of words in a sentence with their opposites and expect it to work every time.

```suggestion
	description = "Bluespace technology using stable and balanced procedures. Required by galactic convention for public use."
```

Reply:
Thank